requirements.txt

```diff
@@ -1,3 +1,9 @@
 numpy>=1.20
 pylibiio==0.23.1
 paramiko
+scipy
+matplotlib
+pyserial
+PyQt5
+pyeit
+pandas
```

Comment:
Please move additional packages to the requirements file inside your examples/cn0565 folder

Reply:
Fixed the dependency list.